platform/local/cluster.go

```diff
@@ -93,6 +93,20 @@ func (lc *LocalCluster) GetOmahaHostPort() (string, error) {
 	return net.JoinHostPort(lc.hostIP(), port), nil
 }
 
+func (lc *LocalCluster) NewTangListener() (*net.Listener, error) {
```

Comment:
This does not seem to be specific at all to Tang, so maybe name it `NewListenerInsideClusterNS`?

Reply:
makes sense - pushed the change